controllers/imageclusterinstall_controller.go

```diff
@@ -340,9 +340,12 @@ func (r *ImageClusterInstallReconciler) validateSeedReconfigurationWithBMH(
 	// no need to validate if inspect annotation is disabled
 	if bmh.ObjectMeta.Annotations != nil && !isInspectionEnabled(bmh) {
 		msg := fmt.Sprintf("inspection is disabled for BareMetalHost %s/%s, skip hardware validation", bmh.Namespace, bmh.Name)
-		r.Log.Info(msg)
-		if updateErr := r.setRequirementsMetCondition(ctx, ici, corev1.ConditionTrue, v1alpha1.HostValidationSucceeded, msg); updateErr != nil {
-			r.Log.WithError(updateErr).Error("failed to update ImageClusterInstall status")
+		condition := findCondition(ici.Status.Conditions, v1alpha1.HostValidationSucceeded)
```

Comment:
Actually i don't think we need this change, this was relevant to previous code version but not for new one,
we will set bmh.Spec.ExternallyProvisioned right after first run in case !isInspectionEnabled(bmh) so this case is not relevant anymore

Reply:
Where does that happen?
AFAIK, [this](https://github.com/openshift/image-based-install-operator/blob/7200975383c1a07fc97b6f60f4b11737b34ec0b9/controllers/imageclusterinstall_controller.go#L613) is the only place where we set the bmh.Spec.ExternallyProvisioned

Reply:
Exactly, this code will run only once cause in case externally provisioned was not set we will set it on the first run. Second time we will be here we will exit 
https://github.com/openshift/image-based-install-operator/pull/102/files#diff-2b950e24497af9d913e379e1aad24d9446d374d42e696b0ee2a0bd5079b3b5a2R331

Reply:
We will only set it once the BMH is available which may take some time ( a lot of time if it goes to error).
Also in you link I can see another spamming log message:
```BareMetalHost ... is externally provisioned, skipping hardware validation```
Fixing that as well

Reply:
Good